board/drivers/llgpio.h

```diff
@@ -14,6 +14,10 @@ void set_gpio_output(GPIO_TypeDef *GPIO, int pin, int val) {
   set_gpio_mode(GPIO, pin, MODE_OUTPUT);
 }
 
+int get_gpio_output(GPIO_TypeDef *GPIO, int pin) {
```

Comment:
not used, I can remove it, right? @legonigel

Reply:
Sure.